data-migration/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "song_5_3_migration",
+  "version": "1.0.0",
+  "description": "Data migration for Song 5.3.x",
+  "main": "src/index.js",
+  "type": "module",
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run build && node dist/index.js",
+    "dev": "tsx src/index.ts",
```

have a feeling this script doesn't need a built out version
```suggestion
    "start": "tsx src/index.ts",
```

---

removed `build` and `start` scripts.